lib/get-new-logger.ts

```diff
@@ -0,0 +1,75 @@
+/**
+ * (C) Copyright IBM Corp. 2019, 2021.
```

Comment:
```suggestion
 * (C) Copyright IBM Corp. 2019, 2022.
```
:joy:

Reply:
Wow, good catch haha

Actually - these are totally new files so shouldn't they just be `2022` ?

Reply:
actually yes :)